style.css

```diff
@@ -1,250 +1,530 @@
+  margin-right: 25px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .menu__link_entrance {
-display: flex;
-align-items: center;
-margin-left: 32px;
+  margin-right: 27px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .header__burger {
-display: flex;
-align-items: center;
-width: 24px;
-height: 17px;
-background: #E51937;
-background: url(./images/Vector.png);
-margin-left: 30px;
-border-right: 0;
-border-left: 0;
-border-bottom: 0;
+  position: absolute;
+  width: 24px;
+  height: 17px;
+  background: #E51937;
+  background: url(./images/Vector.png);
+  border-right: 0;
+  border-left: 0;
+  border-bottom: 0;
+  margin-right: 0;
 } 
 
+.top {
+  display: flex;
+  flex-direction: column;
+}
+
 .top__title {
-display: flex;
-align-items: center;
-width: 370px;
-height: 10px;
-left: 535px;
-font-weight: 600;
-font-size: 90px;
-line-height: 10px;
-color: #000000;
-margin-left: 527px;
-margin-bottom: 40px;
-margin-top: 135px;
+  text-align: center;
+  height: 10px;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 10px;
+  color: #000;
+  margin-right: 142px;
+  margin-bottom: 26px;
 }
 
 .top__subtitle {
-display: flex;
-align-items: center;
-width: 250px;
-height: 16px;
-left: 600px;
-font-weight: normal;
-font-size: 15px;
-line-height: 18px;
-color: #000000;
-margin-left: 590px;
-margin-top: 43px;
-}
-
-.top__avto {
-display: flex;
-align-items: center;
-width: 968px;
-height: 450px;
-margin-left: 190px;
-margin-top: -323px;
+  text-align: center;
+  height: 16px;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  margin-right: 142px;
 }
 
 .top__fon {
-display: flex;
-align-items: center;
-width: 1514px;
-height: 467px;
-background: url("./images/fon.png");
-margin-top: -131px;
-margin-left: -10px;
+ position: inherit;
+ margin-bottom: 40px;
 }
 
-.middle__title {
-color: red;
-font-weight: 600;
-font-size: 90px;
-line-height: 10px;
-margin-left: 100px;
-margin-top:100px;
+.top__avto{
+  max-width: 100%;
 }
+
 .other {
-display: flex;
-justify-content: space-around;
-flex-flow: row nowrap;
-width: 1000px;
-height: 40px;
-margin-left: 200px;
+  display: flex;
+  justify-content: space-around;
+  margin: 0 auto;
+  max-width: 100%;
+  padding: 0 418px 0 270px;
+  padding-bottom: 22px;
+}
+
+.other__block1 {
+  display: flex;
+  align-items: center;
+  margin-right: 5px;
+}
+
+.other__block2 {
+  display: flex;
+  align-items: center;
+  margin-right: 26px;
+}
+
+.other__block3 {
+  display: flex;
+  align-items: center;
+}
+
+.other__speedometr {
+ margin-right: 10px;
+}
+
+.other__speed {
+  
+  height: 44.13px;
+}
+
+.other__text-left {
+  display: flex;
+  flex-direction: column;
+}
+
+.other__left {
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.other__sec {
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.other__bottom-l {
+  
+  font-size: 12px;
+  line-height: 15px;
+  bottom: 17px;
+}
+
+.other__center {
+ margin-right: 10px;
+ margin-top: 10px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)